spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-dead-letter-sample/README.adoc

```diff
@@ -22,15 +22,17 @@ Alternatively, if you have the https://cloud.google.com/sdk/[Google Cloud SDK] i
 3. In the link:src/main/resources/application.properties[application.properties] file, a topic and a dead letter topic are already preconfigured for you.
 The topics will be created in your account if they do not already exist.
 
-4. Run `$ mvn clean install` from the root directory of the project.
+4. The Cloud Pub/Sub service account needs the publisher role to publish dead-lettered messages to the dead letter topic and the subscriber role to forward messages from the subscription to the dead letter topic. Please grant both the roles to the service account you are using by going to the subscription details page on Pantheon.
 
-5. Run `$ mvn clean spring-boot:run` in the root of the code sample
+5. Run `$ mvn clean install` from the root directory of the project.
+
+6. Run `$ mvn clean spring-boot:run` in the root of the code sample
```

Comment:
Don't need to do `clean` again, since it was done in Step 5. Why do you want to split these two steps?

Reply:
This is what it was before, I did not change anything. See this -> https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-dead-letter-sample#:~:text=Run%20%24%20mvn%20clean,the%20app%20running.

Do you still want me to change this ?

Reply:
Never mind; I was confused. Can you remove the `clean` from step 6? If the whole project was just cleaned and rebuilt, it should be unnecessary to clean the sample project again.